git_machete/constants.py

```diff
@@ -27,3 +28,19 @@ class GitFormatPatterns(Enum):
     FULL_MESSAGE = "%B"
     # subject NOT included
     MESSAGE_BODY = "%b"
+
+
+class SquashMergeDetection(Enum):
+    NONE = "none"
+    SIMPLE = "simple"
+    EXACT = "exact"
+
+    @staticmethod
+    def from_string(value: str) -> 'SquashMergeDetection':
+        if value == "none":
+            return SquashMergeDetection.NONE
+        if value == "simple":
```

Comment:
To be covered by tests. There aren't any end-to-end tests currently that would check for a result of a full invocation of `git machete status`, only lower-level tests of `GitContext`. The former are typically preferable.